CHANGELOG_DESKTOP.md

```diff
@@ -769,7 +769,7 @@
  - Removed Brave autoplay exceptions. ([#9597](https://github.com/brave/brave-browser/issues/9597))
  - Removed unnecessary balance API calls from Brave Rewards. ([#9132](https://github.com/brave/brave-browser/issues/9132))
  - Removed retry logic for auto-contribute if auto-contribute or Brave Rewards is disabled. ([#10257](https://github.com/brave/brave-browser/issues/10257))
- - Removed known user tracking parameters from Doubleclick query strings. ([#9879](https://github.com/brave/brave-browser/issues/9879))
+ - Removed known user tracking parameters from Double click query strings. ([#9879](https://github.com/brave/brave-browser/issues/9879))
```

Comment:
This one is not a typo; it's the name of a company actually. Although, the proper capitalization is `DoubleClick` if you wanted to fix